### ISSUE TITLE
#-1928110086------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#492]
<strong>Source:</strong>`<a aria-label="" aria-live="polite" class="button-padding--shrunk icon-end-empty" href="https://velocityautomate.mapsdevext.arcgis.com/home/webmap/viewer.html?layers=31b05d32634046dca7dd571b032a08f4" name="false" rel="noopener noreferrer" target="_blank" type="false">`
<strong>Element Location: </strong>["#map-link-31b05d32634046dca7dd571b032a08f4",".button-padding--shrunk[name=\"false\"][type=\"false\"]"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@19d9ba89
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
